akka-docs-dev/rst/java/code/docs/http/javadsl/server/directives/HostDirectivesExamplesTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package docs.http.javadsl.server.directives;
```

Comment:
how come this file is not in akka-samples/akka-docs-java-lambda? We don't need Java8 stuff?